person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ ] Same here, Please add Getters and Setters (attr_accessor) for https://github.com/rentals, not getters only (attr_reader) as you did so you can create a new rental.

Reply:
I have kindly added the Getters and Setters using attr_accessor